From 4f9621cc16a539e3538bdff03aabf1e03c6bef8f Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:05:07 -0400
Subject: [usb] misc/idmouse: correct spelling mistake in error string

Message-id: <1410887463-20674-236-git-send-email-dzickus@redhat.com>
Patchwork-id: 95553
O-Subject: [RHEL7 PATCH 235/591] USB: misc: idmouse: correct spelling mistake in error string
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 45868b3a98605172ebe6367f602dca347e1d2904
Author: Rahul Bedarkar <rahulbedarkar89@gmail.com>
Date:   Thu Jan 2 19:25:50 2014 +0530

    USB: misc: idmouse: correct spelling mistake in error string

    Signed-off-by: Rahul Bedarkar <rahulbedarkar89@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/misc/idmouse.c b/drivers/usb/misc/idmouse.c
index ce97838..49235bd 100644
--- a/drivers/usb/misc/idmouse.c
+++ b/drivers/usb/misc/idmouse.c
@@ -386,7 +386,7 @@ static int idmouse_probe(struct usb_interface *interface,
  result = usb_register_dev(interface, &idmouse_class);
  if (result) {
   /* something prevented us from registering this device */
-  dev_err(&interface->dev, "Unble to allocate minor number.\n");
+  dev_err(&interface->dev, "Unable to allocate minor number.\n");
   usb_set_intfdata(interface, NULL);
   idmouse_delete(dev);
   return result;
-- 
1.7.1